test/src/node.rs

```diff
@@ -18,6 +18,7 @@ use std::borrow::Borrow;
 use std::collections::HashSet;
 use std::convert::Into;
 use std::fs;
+use std::io::prelude::*;
```

Comment:
Why we need the import of `std::io::preclude::*`? Try to remove it or import what we require only.